### DISCUSSION TITLE
deploy on linux 425 error

### DISCUSSION THREAD

**Discussion by A-little-bit-of-data** [trinodb/trino#18884] [Q&A]
![image](https://github.com/trinodb/trino/assets/31812278/b20ae7f3-8eff-4cf2-8b4a-4d120a6fca9f)
I downloaded the trino-server-425.tar.gz package from the official website and deployed it according to the steps on the official website. The java version used is as follows:
java version "17.0.8" 2023-07-18 LTS
Java(TM) SE Runtime Environment (build 17.0.8+9-LTS-211)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.8+9-LTS-211, mixed mode, sharing)
Several configuration files in the etc directory are as follows

config.properties:
coordinator=true
node-scheduler.include-coordinator=true
http-server.http.port=8080
discovery.uri=http://localhost:8080

jvm.config:
-server
-Xmx16G
-XX:InitialRAMPercentage=80
-XX:MaxRAMPercentage=80
-XX:G1HeapRegionSize=32M
-XX:+ExplicitGCInvokesConcurrent
-XX:+ExitOnOutOfMemoryError
-XX:+HeapDumpOnOutOfMemoryError
-XX:-OmitStackTraceInFastThrow
-XX:ReservedCodeCacheSize=512M
-XX:PerMethodRecompilationCutoff=10000
-XX:PerBytecodeRecompilationCutoff=10000
-Djdk.attach.allowAttachSelf=true
-Djdk.nio.maxCachedBufferSize=2000000
-XX:+UnlockDiagnosticVMOptions
-XX:+UseAESCTRIntrinsics
-Dfile.encoding=UTF-8
####Disable Preventive GC for performance reasons (JDK-8293861)
-XX:-G1UsePreventiveGC

node.properties：
node.environment=test
node.id=1
node.data-dir=/data/trino
#plugin.dir=/data/trino-server-425/plugin

Complete startup error log:
[trino.log](https://github.com/trinodb/trino/files/12483372/trino.log)

Server.log is displayed in the var/log directory.

![image](https://github.com/trinodb/trino/assets/31812278/680b10a7-6866-4bdf-b4cc-cf7465a27c96)

**Comment by hashhar** [comment]
See https://github.com/trinodb/trino/discussions/18841
